fusionfrontend/src/app/factory/components/content/assets-list/assets-list.component.ts

```diff
@@ -114,6 +119,7 @@ export class AssetsListComponent implements OnInit {
   createDetailsAssetForm(formBuilder: FormBuilder) {
```

Comment:
Add // TODO: Has to be extracted into Dialog/AssetInstantiationComponent (IF-429)